build/ci/production-values.yaml

```diff
@@ -194,7 +197,7 @@ cronJobs:
   dailyEmailNotifications:
     enabled: true
   indexFromQueue:
-    enabled: true
+    enabled: false
```

Comment:
This needs to be turned back on?

Reply:
After the main index job completes, yes

Reply:
ah, thought this was the job that has the temporary override